assets/prototype/domain/eventEditor/data/queries/prices/useFetchPrices.ts

```diff
@@ -6,10 +6,13 @@ import usePriceQueryOptions from './usePriceQueryOptions';
 import { FetchEntitiesResult } from '../types';
 import { PricesList } from '../../types';
 
-const useFetchPrices = (): FetchEntitiesResult<PricesList> => {
-	const { setIsLoading, setIsLoaded, setIsError } = useStatus();
+const useFetchPrices = (skipFetch: boolean = undefined): FetchEntitiesResult<PricesList> => {
```

Comment:
maybe `null` is better ?
my understanding is that `null` is used when it is being set by dev and `undefined` is a result of a processing or an operation

Reply:
Agreed 👍

Reply:
Changed